### ISSUE TITLE
Add "Sort by" filter for clipboard entries in clipboard UI

### ISSUE THREAD

**Issue by Donkey-Doug** [florisboard/florisboard#1373]
A lot of the time when a I copy a new word and move to the clipboard, I cannot see the entry. For quite a long time I assumed it was a bug, but apperently I need to swipe up to reveal new entries when I already have some entries in my history.

**Comment by Glitchy-Tozier** [comment]
I think the clipboard entries should actually get ordered most recent to oldest. 

I guess currently it's the other way around.

**Comment by Donkey-Doug** [comment]
Yes indeed, that is the problem I tried to describe.

**Comment by patrickgold** [comment]
Thanks for your feature request!

IMO a "Sort by" filter would be best, then everyone can have it the preferred way. Will add it to the list of clipboard issues.

**Comment by Glitchy-Tozier** [comment]
>I think the clipboard entries should actually get ordered most recent to oldest.

@patrickgold I'd still like to point oth that imo this ist the most useful default filtering-mode.

**Comment by patrickgold** [comment]
> I'd still like to point oth that imo this ist the most useful default filtering-mode.

I agree with this one, when the clipboard UI gets the filter that will be the default, that's what I also thought when I replied yesterday (but forgot to specify it in words).

**Comment by lm41** [comment]
Superseded by #2925
